### PR TITLE
Changing the quickstart doc according to issue #15.

### PR DESCRIPTION
**Pull request by lljrsr**
These are the changes that I mentioned. I hope they are okay this way.

### CODE REVIEW

File: docs/source/topics/quickstart.rst [hunk 1]
```diff
@@ -24,6 +25,11 @@ For Ubuntu, type in command line: ::
 
     $ pip install distributed-frontera
 
+Create a namespace in the HBase shell for the crawler: ::
```
Comment (sibiryakov) [inline docs/source/topics/quickstart.rst]: That one definitely needs to be here!

File: docs/source/topics/quickstart.rst [hunk 2]
```diff
@@ -14,8 +14,9 @@ Prerequisites
 Here is what services needs to be installed and configured before running Frontera:
 
 - HBase.
+- Thrift with support for Python
 
-These can be set up locally, no specific tuning is needed.
+These can be set up locally in pseudo-distributed mode, no specific tuning is needed.
```
Comment (sibiryakov) [inline docs/source/topics/quickstart.rst]: Did you tried ``Standalone mode`` https://hbase.apache.org/book.html#standalone_dist ? It works on my developer machine for months.

Comment (lljrsr) [inline docs/source/topics/quickstart.rst]: Actually you are right. Using it in Standalone mode also works. Sorry.

Comment (sibiryakov) [inline docs/source/topics/quickstart.rst]: I'm going to merge this PR, please update the branch, excluding this.

**Comment by lljrsr** [comment]
Done :)

File: docs/source/topics/quickstart.rst [hunk 3]
```diff
@@ -13,7 +13,8 @@ Prerequisites
 
 Here is what services needs to be installed and configured before running Frontera:
 
-- HBase.
+- HBase
+- Thrift with support for Python
```
Comment (sibiryakov) [inline docs/source/topics/quickstart.rst]: Is there anything specific? As I know, HBase comes with bundled Thrift server, and ``distributed-frontera`` has ``happybase`` which depends on Thrift for Python.

Comment (lljrsr) [inline docs/source/topics/quickstart.rst]: I will set up a clean install of HBase and will tell you soon.

Comment (lljrsr) [inline docs/source/topics/quickstart.rst]: You are right. My mistake. Still.. maybe I could add a line telling people to start the Thrift server before running Frontera?

Comment (sibiryakov) [inline docs/source/topics/quickstart.rst]: That would be great!

File: docs/source/topics/quickstart.rst [hunk 4]
```diff
@@ -38,7 +43,7 @@ of predefined options, please consult settings reference to get more information
 Start cluster
 =============
 
-First, let's start ZeroMQ broker. ::
+Make sure that the HBase Thrift server is up and running. First, let's start ZeroMQ broker. ::
```
Comment (sibiryakov) [inline docs/source/topics/quickstart.rst]: Next sentence sounds a bit strange. May be 
> First, make sure that the HBase Thrift server is up and running. Next, let's start ZeroMQ broker.

**Pull request merged**